woocommerce/woocommerce
# Add button on toolbar to toggle show/hide block inspector

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Add button on product description toolbar to toggle show/hide block inspector

<img width="1124" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13437655/003ceefc-c84a-4014-963d-1714970f8cd2">

<img width="1115" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13437655/b7764d2c-0664-4967-b5e4-1d6e96d42dd4">




Closes #39025 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Enable "Try the new product editor (Beta)" in WooCommerce > Advanced > Features.
2. Go to Product > Add New.
3. Click "Add Description".
4. Check the button on the top right of the modal.
5. Click it and check if it shows/hides the block inspector.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
